addons/lunch/wizard/lunch_confirm.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, api
+
+
+class lunch_confirm_order(models.TransientModel):
```

Comment:
We don't prompt the wizard on every action! The wizard is only used through the "More" menu, in order to Order / Confirm / Cancel multiple orders at a time.